### PR TITLE
Mooderald Tools

### PR DESCRIPTION
**Pull request by kaosj314**
Mooderald sword: Costs 8192+6 Mooderald
           7-11 damage, increases emerald gain from mob loot by 1.1
Mooderald Pickaxe: Costs 8192+4 Mooderald
           50% emerald chance

### CODE REVIEW

File: src/cogs/cmds/econ.py [hunk 1]
```diff
@@ -1044,7 +1044,7 @@ async def mine(self, ctx):
 
             # only works cause num of pickaxes is 6 and levels of fake finds is 3
             # please don't bug me about jank code, I know
-            fake_finds = self.d.mining.finds[math.floor(self.d.mining.pickaxes.index(pickaxe) / 2)]
+            fake_finds = min(self.d.mining.finds[math.floor(self.d.mining.pickaxes.index(pickaxe) / 2)],3)
```
Comment (Iapetus-11) [inline src/cogs/cmds/econ.py:1047]: What is this supposed to do?

File: src/cogs/other/mobs.py [hunk 2]
```diff
@@ -56,7 +56,9 @@ def attack_check(self, m, e_msg, ctx):
 
     async def calc_sword_damage(self, uid, sword, diff_multi):
         sword = sword.lower()
-
+        
+        if sword == "mooderald sword":
+            dmg = random.randint(7,11)
         if sword == "netherite sword":
```
Comment (Iapetus-11) [inline src/cogs/other/mobs.py]: You'd need to use elif here

File: src/cogs/other/mobs.py [hunk 3]
```diff
@@ -307,6 +309,8 @@ async def spawn_event(self, ctx):
 
                     ems_won = int((ems_won if ems_won > 0 else 1) * diff_multi)
 
+                    if await self.db.fetch_item(u.id, "Mooderald Sword") is not None:
+                        ems_won = int(ems_won * 1.1)
```
Comment (Iapetus-11) [inline src/cogs/other/mobs.py:313]: add a newline after this if statement if it's not supposed to be part of the if/elif chain below it.

**Review by Iapetus-11 — CHANGES_REQUESTED** [review]

File: src/cogs/other/mobs.py (hunk 3 above)
Comment (kaosj314) [reply src/cogs/other/mobs.py:313]: oh oops

File: src/cogs/cmds/econ.py (hunk 1 above)
Comment (kaosj314) [reply src/cogs/cmds/econ.py:1047]: so that the fake finds aren't broken by pickaxe number 7 in a function built for 6 pickaxes

Comment (Elrynd) [reply src/cogs/cmds/econ.py:1047]: That will break, finds[3] will be out of range, that should be min(<some stuff>, 2) if you want that to work

**Comment by Iapetus-11** [comment]
tbh mooderalds weren't intended to be used for tools

File: src/cogs/cmds/econ.py (hunk 1 above)
Comment (kaosj314) [reply src/cogs/cmds/econ.py:1047]: oh thanks. I must have some math wrong

**Comment by Elrynd** [comment]
It wasn't intendend to be a secondary currency used for auction either
It was nothing but a meme
So why not make it a tool too, could give more life and value to mining ?

**Pull request closed without merging**